src/App.js

```diff
@@ -54,11 +60,35 @@ class App extends Component {
       });
   
       this.setState({
+        currentUsStateName: 'default',
+        currentUsStateCoord: [],
         currentParksToShow: bucketListParks
       });
     } 
   }
 
+  openHomePage = () => {
+    this.setState({
+      pageStatus: 'home'
+    });
+  }
+
+  setMapToState = (stateName, stateObj) => {
```

Comment:
This is great! Thank you for adding in functionality to show only the parks that are in the current selected state (if a state is selected from Filter controls).